src/main/java/com/targus/ui/Controller.java

```diff
@@ -236,14 +236,18 @@ protected Solution call() {
             HashSet<Integer> indexes = bitString.ones();
 
             WSNMinimumSensorObjective wsnMinimumSensorObjective = new WSNMinimumSensorObjective();
-            double sensorPenValueScaled = wsn.getSolutionSize() != 0 ?
-                    1 - ((double) bitString.getBitSet().cardinality() / wsn.getSolutionSize()) : 0;
-
-            double mConnPenValueScaled = indexes.size() == 0 || wsn.getM() == 0 ?
-                    1 : (double) wsnMinimumSensorObjective.mConnPenSum(wsn, indexes) / (indexes.size() * wsn.getM());
-
-            double kCoverPenValueScaled = wsn.targetsSize() * wsn.getK() != 0 ?
-                    (double) wsnMinimumSensorObjective.kCovPenSum(wsn, indexes) / (wsn.targetsSize() * wsn.getK()) : 1;
+//            double sensorPenValueScaled = wsn.getSolutionSize() != 0 ?
```

Comment:
This commented out code can be deleted, however I've decided to keep it just in case the separate methods conception is not endorsed.